ui/page/wallet_page.go

```diff
@@ -775,6 +780,12 @@ func (pg *WalletPage) layoutAddWalletSection(gtx layout.Context) layout.Dimensio
 	})
 }
 
+func (pg *WalletPage) clickableRow(gtx layout.Context, clickable *widget.Clickable, body layout.Widget) layout.Dimensions {
```

Comment:
Nothing in this function makes it a 'clickable row', it's just a wrapper for decredmaterial clickable. It should be removed and decredmaterial clickable should be used directly.